### PR TITLE
Ch 3328: Help text

### PR DESCRIPTION
**Pull request by eshtadc**
Mostly text changes.
To note: goals that are created automatically now have a property set in the visitor action data.
Also - we are tracking whether a user dismissed a section help in the user data property.

### CODE REVIEW

File: acquia_lift.module [hunk 1]
```diff
@@ -894,7 +899,9 @@ function acquia_lift_personalize_option_set_insert($option_set) {
     'identifier' => $selector,
     'event' => 'click',
     'pages' => $pages,
-    'data' => array(),
+    'data' => array(
+      'auto_created' => $option_set->osid,
+    ),
```
Comment (katbailey) [inline acquia_lift.module:904]: That is pretty innocuous :)

File: acquia_lift.admin.wizard.inc [hunk 2]
```diff
@@ -1321,6 +1498,20 @@ function acquia_lift_personalize_campaign_wizard_review_alter(&$form, &$form_sta
...
+ * Ajax submit handler for any help section dismissal button
+ */
+function acquia_lift_personalize_campaign_wizard_help_ajax($form, &$form_state) {
+  global $user;
+  $current_dismiss = !empty($user->data['acquia_lift_help_dismiss']) ? $user->data['acquia_lift_help_dismiss'] : array();
+  // #name property is "display_{step}"
+  $dismiss = preg_replace('/^dismiss_/', '', $form_state['triggering_element']['#name']);
+  if (!in_array($dismiss, $current_dismiss)) {
+    $current_dismiss[] = $dismiss;
+    user_save($user, array('data' => array('acquia_lift_help_dismiss' => $current_dismiss)));
```
Comment (katbailey) [inline acquia_lift.admin.wizard.inc:1510]: I might be reading this wrong... couldn't we be clobbering other stuff in the `$user->data` property here?

Comment (katbailey) [inline acquia_lift.admin.wizard.inc:1510]: Oh, nvm... just checked user_save and I see it gets array_merged in... all good.

Comment (eshtadc) [inline acquia_lift.admin.wizard.inc:1510]: Exactly - I checked that too.

**Pull request closed without merging**